### DISCUSSION TITLE
(off-topic) Did anyone know how can I restore the smart bookmarks folers?

### DISCUSSION THREAD

**Discussion by Aleksandra0056** [black7375/Firefox-UI-Fix#653] [General]
I'm writing this post here, because one guy have the smart bookmarks folders in screenshot with the Firefox UI Fix installed. 
But how can I restore the smart folders like: Recent tags, Recently bookmarked, Most visited. Are there any Firefox modifications to add the ability to restore them? Or the guy have older version of Firefox in that screenshot?

Here I'm attaching screenshot with smart bookmarks folders:
![image](https://user-images.githubusercontent.com/69351811/220812905-12078e1a-8d3d-46dd-b0ca-f899a9279bf7.png)

**Comment by black7375** [comment]
There is already a [document in this repo](https://github.com/black7375/Firefox-UI-Fix/blob/master/docs/Smart_Bookmarks.md).

**Reply by Aleksandra0056** [reply]
But how can I type in the folder/bookmark “location” field? The field disappears immediately when I'm going to create a new folder/bookmark.

**Reply by black7375** [reply]
1. Create Smart Bookmark
![image](https://user-images.githubusercontent.com/25581533/220987861-87658dcc-ce0b-4eff-b1e4-7a72de0532c5.png)

2. Restart, just work for me
![image](https://user-images.githubusercontent.com/25581533/220987594-745e07d4-9ecc-4398-8d13-860623021b10.png)

**Reply by Aleksandra0056** [reply]
Thank you, solved.

**Reply by Aleksandra0056** [reply]
And how can I create recent tags?

**Reply by black7375** [reply]
Maybe `type` and `sort` in the document will be helpful.

**Comment by Aleksandra0056** [comment]
Fine, I'm closing this discussion. Thank you all for help!
